tests/dummy/app/routes/nodes/detail/draft-registrations/index.js

```diff
@@ -8,6 +8,8 @@ export default Ember.Route.extend({
     },
     setupController(controller, model) {
         this._super(controller, model);
+        var metaschemas = this.store.findAll('metaschema');
+        controller.set('metaschemas', metaschemas);
```

Comment:
One crude mechanism would be to only set the controller property after the promise was resolved, inside `.then`